src/index.ts

```diff
@@ -19,9 +19,10 @@ export interface PrometheusMetricsInit {
   collectDefaultMetrics?: boolean
 
   /**
-   * prom-client options to pass to the `collectDefaultMetrics` function
+   * prom-client options to pass to the `collectDefaultMetrics` function.
+   * Set to `false` to disable `collectDefaultMetrics`.
    */
-  defaultMetrics?: DefaultMetricsCollectorConfiguration
+  defaultMetrics?: DefaultMetricsCollectorConfiguration | false
```

Comment:
why can this be false. Can't you pass `{collectDefaultMetrics: false, defaultMetrics: undefined}`?

Reply:
Look at the conditional on L44 and L45 (the other diff here)

This was an attempt to do a backwards-compatible change. Current behavior is to pass `opts.defaultMetrics` (undefined or not) to `collectDefaultMetrics`

Reply:
It seems like L44 is a bug. Why don't we use option from L19 on L44?